test-helpers/src/connection/kafka/mod.rs

```diff
@@ -164,6 +165,28 @@ impl KafkaConsumer {
             }
         }
     }
+
+    /// The offset to be committed should be lastProcessedMessageOffset + 1.
+    pub fn assert_commit_offsets(&self, offsets: HashMap<TopicPartition, i64>) {
```

Comment:
I think this method should be async, if the integration tests block then shotover logs are also blocked which can make debugging tricky.
I think the best way to solve this is to call within the cpp/java consumers https://docs.rs/tokio/latest/tokio/task/fn.block_in_place.html so that we can call the blocking functions without blocking the async runtime.

For example we already do that here:
https://github.com/shotover/shotover-proxy/blob/6ec2022ff303c553fd183ffeef9204730d2c44ae/test-helpers/src/connection/kafka/java.rs#L212

Reply:
Cool will change it